python/openmldb/test/openmldb_client_test.py

```diff
@@ -395,6 +395,55 @@ def test_drop_table(self):
 
     def teardown_class(self):
         self.cursor.close()
+        
+class TestSQLMagicOpenMLDB:
+
+    def setup_class(self):
+        self.db = openmldb.dbapi.connect('db_test', '127.0.0.1:6181', '/onebox')
+        self.ip = openmldb.sql_magic.register(self.db)
+
+    def execute(self,magic_name,line):
```

Comment:
Since our magic function is always "sql", you can remove the this parameter and only pass the `line`.

The parameter `line` would change to be `sql` or `code` which is more meaningful.